docs/dev/libvirt/README.md

```diff
@@ -151,6 +151,22 @@ so we only need to allow traffic on the libvirt port:
 sudo firewall-cmd --zone=libvirt --add-service=libvirt
 ```
 
+*NOTE:* The installer creates a libvirt network and a `tt0` bridge.
+This bridge name can be customized in the `install-config.yaml` file in the
+`platform.libvirt.network.if` section as:
+
+```
+apiVersion: v1beta1
+baseDomain: example.com
+machines:
+...
+platform:
+  libvirt:
+    URI: qemu+tcp://192.168.122.1/system
+    network:
+      if: mybridge0
+```
+
```

Comment:
I'm not certain we need this note here for only one part of the `install-config`. Why not add a more generic note which lists essential items one can configure (including the bridge name)?

Reply:
Like in a different section you mean?

Reply:
I guess it could be a separate section. Just think that all aspects of `install-config.yaml` are important, not only the `tt0` part.

Reply:
I agree on having all the `install-config.yaml` parameters documented, but I believe this goes beyond this PR (it will deserve a whole doc by itself if you ask me)

Reply:
a PR's purpose can be expanded as needed. :) I'm not expecting you to document all parameters but document this so that it's clear for people in the future where to add docs for other parameters.

Reply:
I did opened https://github.com/openshift/installer/issues/2058 in order to have a proper customization doc.

Reply:
Right, and based on @abhinavdahiya 's comment there, it seems you just need a similar customization doc for libvirt then.

Reply:
Ok, I've just force pushed a new customization.md doc with the `tt0` and a note in the README.